pkg/config/user_config.go

```diff
@@ -504,7 +504,8 @@ type KeybindingSubmodulesConfig struct {
 }
 
 type KeybindingCommitMessageConfig struct {
-	CommitMenu string `yaml:"commitMenu"`
+	CommitMenu         string `yaml:"commitMenu"`
+	PasteFromClipboard string `yaml:"pasteFromClipboard"`
```

Comment:
> I'm confused, this doesn't actually seem to be used anywhere.

Yeah it's not actually used, and I don't want it to be configurable.

This field was added because the PR requirements above mentiond docs, so
I added a field here for docs.

Maybe I put it in a wrong place? But I'll remove it if it's unnecessary.

Reply:
I see. In that case I'm in favour of removing the commit, it doesn't make sense to add the config without using it.

We might eventually want to make these keys customizeable; I could imagine some people could prefer `v` (or even `<c-v>`) for pasting. But we don't have to do this as part of this PR.